politeiad/backendv2/tstorebe/plugins/dcrdata/dcrdata.go

```diff
@@ -0,0 +1,694 @@
+// Copyright (c) 2020-2021 The Decred developers
```

Comment:
This file needs doco on functions.

Reply:
Done

https://github.com/lukebp/politeia/blob/tlog/politeiad/backendv2/tstorebe/plugins/dcrdata/cmds.go